develop/plone/content/importexport.rst

```diff
@@ -37,7 +37,7 @@ To export objects from a site to another, do the following:
 
 * Zope .zexp to *youranothersite*/var/instance/import folder
 
-* Go to ZMI root of your another site
+* Go to the Management Interface root of your another site
```

Comment:
"your another" should either be "your other" or just "another" (not sure of the context)